build/Targets/References.targets

```diff
@@ -117,7 +117,10 @@
 		if it is not provided here, tests fail to load this assembly at runtime.
 	-->
         <Reference Include="Microsoft.VisualStudio.Diagnostics.PerformanceProvider">
-          <HintPath>$(DevEnvDir)\PrivateAssemblies\Microsoft.VisualStudio.Diagnostics.PerformanceProvider.dll</HintPath>
+          <HintPath Condition="Exists('$(DevEnvDir)\PrivateAssemblies\Microsoft.VisualStudio.Diagnostics.PerformanceProvider.dll')">$(DevEnvDir)\PrivateAssemblies\Microsoft.VisualStudio.Diagnostics.PerformanceProvider.dll</HintPath>
```

Comment:
It might be cleaner to do this sort of checking at the place that is setting DevEnvDir, so all consumers of it are correct. But this is fine.